src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlag.cs

```diff
@@ -15,5 +16,14 @@ internal class FeatureFlag
 
         [JsonPropertyName("conditions")]
         public FeatureConditions Conditions { get; set; }
+
+        [JsonPropertyName("variants")]
+        public List<FeatureVariant> Variants { get; set; }
```

Comment:
Does this need to be List?

Reply:
I was mostly doing this since we use List in the existing `FeatureConditions` file, but looks like the code could be changed to use IEnumerable instead. Was that what you were suggesting?